pages/contribution/_slug.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div>
+    <carbon-ads :key="$route.params.slug" />
+    <html-parser :content="body" />
+    <contributor-list />
+    <contribute :doc-link="docLink" />
+  </div>
+</template>
+
+<script>
+import axios from 'axios'
+import CarbonAds from '~/components/CarbonAds.vue'
+import ContributorList from '~/components/ContributorList.vue'
+import HtmlParser from '~/components/HtmlParser.vue'
+import Contribute from '~/components/Contribute.vue'
+
+export default {
+  async asyncData({ route, store, error }) {
+    // Default data
+    let data = {
+      attrs: {},
+      body: '',
+      docLink: ''
+    }
+    const slug = route.params.slug || 'contributors'
+    const path = `/${store.state.lang.iso}/contribution/${slug}`
+    let res
+    try {
+      res = await axios.get(store.state.apiURI + path)
+    } catch (err) {
+      if (err.response.status !== 404) {
+        return error({ statusCode: 500, message: store.state.lang.text.an_error_occured })
+      }
+      return error({ statusCode: 404, message: store.state.lang.text.api_page_not_found })
+    }
+    data.attrs = res.data.attrs
+    data.body = res.data.body
+    data.docLink = `https://github.com/nuxt/docs/blob/master${path}.md`
+    if (store.state.lang.iso === 'ru') {
+      data.docLink = `https://github.com/translation-gang/ru.docs.nuxtjs/blob/translation-ru${path}.md`
```

Comment:
Can we make it more dynamic? Maybe an object from API...

Reply:
Actually, this is the copy from the following like that and I don't know why ru and cn are special.

* https://github.com/inouetakuya/nuxtjs.org/blob/df6c19d2980b45dd610a37188f81dc5a2d375a52/pages/guide/_slug.vue#L40-L45

I have no good idea and I hope someone will refactor this in another pull request :sweat_smile: